pages.ml/linux/bluetoothctl.md

```diff
@@ -3,7 +3,7 @@
 > കമാൻഡ് ലൈനിൽ നിന്ന് ബ്ലൂടൂത്ത് ഉപകരണങ്ങൾ മാനേജുചെയ്യുക.
 > കൂടുതൽ വിവരങ്ങൾ: <https://www.npmjs.com/package/bluetoothctl>.
 
-- ബ്ലൂടൂത്ത്സിറ്റിഎൽ ഷെല്ലിൽ കേറാൻ 
+- ബ്ലൂടൂത്ത്സിറ്റിഎൽ ഷെല്ലിൽ കേറാൻ
```

Comment:
I can't remember. Do we require colons at the end of examples in this language?

Reply:
Yep
```suggestion
- ബ്ലൂടൂത്ത്സിറ്റിഎൽ ഷെല്ലിൽ കേറാൻ:
```